e2e/steps_file.js

```diff
@@ -146,9 +146,10 @@ module.exports = function () {
       await this.completeEvent('Save and continue');
       await caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
       uploadDocumentsEventPage.selectSocialWorkChronologyToFollow(config.testFile);
-      uploadDocumentsEventPage.uploadSocialWorkStatement(config.testFile);
+      uploadDocumentsEventPage.uploadSocialWorkStatement();
```

Comment:
Could we state in method name that this social work statement is in SWET? (as in the line above above)

Reply:
Yeah definitely let's do this!